### ISSUE TITLE
Clicking on the empty row returns an error (`click-row.bs-table`)

### ISSUE THREAD

**Issue by iTouchTheSky** [wenzhixin/bootstrap-table#1040]
Hi just found out that clicking an empty row will give you an error on `click-row.bs.table` event.
Let's just say I try to do :
```javascript
myTable.on('click-row.bs.table', function(row, data) {
    //do anything here
});
```
It will give this error when trying to click the row that shows there's no result :
`Uncaught TypeError: Cannot read property 'nom' of undefined` (Where `nom` is one of the columns values)

That error is happening in bootstrap-table's code (around line 1407 of the uncompressed version for me) :
```javascript
    // click to select by column
        this.$body.find('> tr > td').off('click').on('click', function () {
            var $td = $(this),
                $tr = $td.parent(),
                item = that.data[$tr.data('index')],
                cellIndex = $td[0].cellIndex,
                $headerCell = that.$header.find('th:eq(' + cellIndex + ')'),
                field = $headerCell.data('field'),
                value = item[field]; //ERROR HAPPENS HERE <-----------------------------------
            that.trigger('click-cell', field, value, item, $td);
            that.trigger('click-row', item, $tr);
            // if click to select - then trigger the checkbox/radio click
            if (that.options.clickToSelect) {
                if (that.header.clickToSelects[$tr.children().index($(this))]) {
                    $tr.find(sprintf('[name="%s"]',
                        that.options.selectItemName))[0].click(); // #144: .trigger('click') bug
                }
            }
        }); 
```

If you need a jsFiddle let me know, but I think you can really easily try this on your own.

**Comment by djhvscf** [comment]
Please provide fiddle please.. That will save our time

**Comment by iTouchTheSky** [comment]
I will do that later as I'm currently working

**Comment by iTouchTheSky** [comment]
I'll try to compare my fiddle to our project at my job when I get back to it in two days, because my fiddle doesn't make it, but I did nothing particular for our project ... I'll check that out and come back to you.

**Comment by wenzhixin** [comment]
This bug has been fixed: #970.

**Comment by iTouchTheSky** [comment]
Didn't saw this as changelog was not up to date, but still it's not in the dist files. Taking src files for now ...
